tests/azure/availset_test.go

```diff
@@ -19,33 +19,41 @@ func testAvailSetCreate(t *testing.T, f fixture.F) {
 	nash.Run(
 		f.Ctx,
 		t,
+		f.Logger,
 		"./testdata/create_avail_set.sh",
 		f.ResGroupName,
 		availset,
 		f.Location,
 	)
-	availSets := azure.NewAvailSet(f.Ctx, t, f.Session, f.ResGroupName)
+	availSets := azure.NewAvailSet(f.Ctx, t, f.Session, f.Logger, f.ResGroupName)
 	availSets.AssertExists(t, availset)
 }
 
 func testAvailSetDelete(t *testing.T, f fixture.F) {
-
 	availset := genAvailSetName()
 	nash.Run(
```

Comment:
@tiago4orion I tried something from the sense that I got from the too much parameters. There was also another problem, the same parameters where being passed over and over again. I think it is better now, take a look ;-)